src/main/java/server/router/routes/RouteTemplate.java

```diff
@@ -0,0 +1,91 @@
+package server.router.routes;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import server.constants.Status;
+import server.request.Request;
+import server.response.Response;
+import server.response.ResponseBuilder;
+
+public abstract class RouteTemplate {
```

Comment:
What is the responsibility of this component?

Reply:
lol I just realized that I don't need the Route class inheriting from the RouteTemplate anymore and can just combine them. Will update now